### ISSUE TITLE
'compareset' error when using trimAL 1.2 on Windows

### ISSUE THREAD

**Issue by dongzhang0725** [scapella/trimal#54]
Dear developer,

I experienced an error while using trimAL. When I using "trimal -compareset Api0000040.cmp" command. I got an error: "Alignment not loaded: "" Check the file's content.". I also tried to use the absolute path of the alignment, but I still got that error. Could you please tell me how to resolve this? I am using trimAl 1.2 on Windows (downloaded from http://trimal.cgenomics.org/downloads).

By the way, could you please compile a new version of trimAL for windows? I can currently only use version 1.2 on Windows. 

Best wishes,

Dong Zhang

**Comment by virologist** [comment]
![image](https://github.com/inab/trimal/assets/54756814/040448c5-5793-40e8-86a0-0ac5430a1cf8)
I got the same error

**Comment by virologist** [comment]
I think I got it!

Here is my experiences.
1. Windows version 1.2 fails when testing `-compareset`. 
2. Therefore, I download and extract the source files for v1.4, compile with make in a Linux environment, and then copy `trimal` and `readal` to the bin directory or another execution directory. Under Linux environment, the `filesetlist.txt` file must use Unix character format; 
3. the MAS files' name can not contain space characters, such as "_", otherwise you will encounter "`Alignment not loaded: "" Check the file's content. file's content.?`" error. 
4. The datasets to be compared must have the same number of sequences, otherwise the error "`ERROR: The files to compare do not have the same number of sequences`" will occur. Using mafft and masce results in different datasets, while masce results in reordering of sequences, the results of the two methods are not compatible and cannot be used together as input for comparison.

```
$ ./trimal -h
trimAl v1.4.rev15 build[2013-12-17]. 2009-2013. Salvador Capella-Gutierrez and Toni Gabaldón.

$ ls *.fasta
globalpair100.fasta  mafft.fasta

$ cat filesetlist.txt
mafft.fasta
globalpair100.fasta

$ ./trimal -compareset filesetlist.txt -out optimal_alignment
File:           mafft.fasta
Values:         Sequences: 2365 Residues: 1507  Pond. Hits:  1449.62    %Consistency: 0.961923

File:           globalpair100.fasta
Values:         Sequences: 2365 Residues: 1507  Pond. Hits:  1450.15    %Consistency: 0.962274
                                        --------------
File Selected:  globalpair100.fasta
Value:          0.962274
```

**Comment by nicodr97** [comment]
Hi,

Thank you for the explanation. I just uploaded the [Windows binaries for version 1.4](https://github.com/inab/trimal/releases/tag/v1.4.1). Moreover, I think the errors that arose with `-compareset` might be solved in the last version of the [main branch](https://github.com/inab/trimal/tree/trimAl). It would be great if you could try it and give me some feedback.

**Comment by nicodr97** [comment]
Solved by #99 and new release [v1.5](https://github.com/inab/trimal/releases/tag/v1.5.0)

**Comment by NoorulemanAbid** [comment]
> Hi,
> 
> Thank you for the explanation. I just uploaded the [Windows binaries for version 1.4](https://github.com/inab/trimal/releases/tag/v1.4.1). Moreover, I think the errors that arose with `-compareset` might be solved in the last version of the [main branch](https://github.com/inab/trimal/tree/trimAl). It would be great if you could try it and give me some feedback.
